src/AstrometryFit.cc

```diff
@@ -549,66 +549,96 @@ void AstrometryFit::checkStuff() {
     std::string separator = "\t";
-    ofile << "#id" << separator << "xccd" << separator << "yccd " << separator << "rx" << separator << "ry"
-          << separator << "xtp" << separator << "ytp" << separator << "mag" << separator << "mjd" << separator
-          << "rvx" << separator << "rvy" << separator << "rvxy" << separator << "color" << separator
-          << "fsindex" << separator << "ra" << separator << "dec" << separator << "chi2" << separator << "nm"
-          << separator << "chip" << separator << "visit" << std::endl;
-    ofile << "#id in source catalog" << separator << "coordinates in CCD" << separator << separator
-          << "residual in degrees in TP" << separator << separator << "transformed coordinate in TP"
-          << separator << separator << "rough mag" << separator << "Modified Julian date of the measurement"
-          << separator << "transformed measurement uncertainty" << separator << separator << separator
-          << "currently unused" << separator << "unique index of the fittedStar" << separator
-          << "on sky position of fittedStar" << separator << separator << "contribution to Chi2 (2D dofs)"
-          << separator << "number of measurements of this fittedStar" << separator << "chip id" << separator
-          << "visit id" << std::endl;
+
+    ofile << "#id" << separator << "xccd" << separator << "yccd " << separator;
+    ofile << "rx" << separator << "ry" << separator;
+    ofile << "xtp" << separator << "ytp" << separator;
+    ofile << "mag" << separator << "mjd" << separator;
+    ofile << "xErr" << separator << "yErr" << separator << "xyCov" << separator;
+    ofile << "xtpi" << separator << "ytpi" << separator;
+    ofile << "rxi" << separator << "ryi" << separator;
+    ofile << "color" << separator << "fsindex" << separator;
+    ofile << "ra" << separator << "dec" << separator;
+    ofile << "chi2" << separator << "nm" << separator;
+    ofile << "chip" << separator << "visit" << std::endl;
+
+    ofile << "#id in source catalog" << separator << "coordinates in CCD (pixels)" << separator << separator;
+    ofile << "residual on TP (degrees)" << separator << separator;
+    ofile << "transformed coordinate in TP (degrees)" << separator << separator;
+    ofile << "rough magnitude" << separator << "Modified Julian Date of the measurement" << separator;
+    ofile << "transformed measurement uncertainty (degrees)" << separator << separator << separator;
+    ofile << "as-read position on TP (degrees)" << separator << separator;
+    ofile << "as-read residual on TP (degrees)" << separator << separator;
+    ofile << "currently unused" << separator << "unique index of the fittedStar" << separator;
+    ofile << "on sky position of fittedStar" << separator << separator;
+    ofile << "contribution to Chi2 (2D dofs)" << separator << "number of measurements of this fittedStar"
+          << separator;
+    ofile << "chip id" << separator << "visit id" << std::endl;
```

Comment:
Are there any non-formatting changes here? I honestly can't tell.

Reply:
Unfortunately, I made the formatting changes after I'd started trying to add things to the output, in order to make the additions/modifications tractable (`clang-format` requires the `ofile` before each line, otherwise it rewraps the chained `<<`). Not sure if it's worth breaking that up, nor if there's a good way to do so.